source/WanManager/wanmgr_dhcpv6_apis.c

```diff
@@ -1484,12 +1484,17 @@ int dhcpv6_assign_global_ip(char * prefix, char * intfName, char * ipAddr)
         CcspTraceError(("error, there is not '::' in prefix:%s\n", prefix));
         return 1;
     }
-#ifdef _HUB4_PRODUCT_REQ_
-    if(strncmp(intfName, COSA_DML_DHCPV6_SERVER_IFNAME, strlen(intfName)) == 0)
+#if defined(_HUB4_PRODUCT_REQ_) || defined(_RDKB_GLOBAL_PRODUCT_REQ_)
+#if defined(_RDKB_GLOBAL_PRODUCT_REQ_)
+    if ( TRUE == WanMgr_Util_IsFeatureApplicable(SYSEVENT_FEATURE_DISABLE_IPV6_EUI64_FORMAT_SUPPORT, INPUT_SOURCE_TYPE_SYSEVENT) )
```

Comment:
Would be possible to use a configuration in PSM instead of depending of a sysevent?

Reply:
Can we introduce this PSM member when we are unifying this? If we introduce new PSM then we need to consider image upgrade use case for existing products so all the default configuration member needs to be update. As per the current PSM migration code, protected under GLOBAL product requirement flag only not for other products.

Please Advise.

Reply:
Addressed all your comments as part of latest update. Could you please check?